### DISCUSSION TITLE
EKS?

### DISCUSSION THREAD

**Discussion by CODY-RAY** [kubernetes-sigs/gateway-api#938] [General]
I'm on gke and I see a lot of docs for this api on gke but I can't find docs on aws and I don’t have access to a EKS cluster. 
So I was wondering how well this will be supported on other kinds of clusters

**Comment by howardjohn** [comment]
There are generally 2 types of implementations:
* In cluster ones, which I assume would all work on any conformant k8s cluster, but I cannot speak for all of them. I know Istio's would work
* Cloud ones, which use the cloud load balancing infra. I only know of GKE that supports this today, but would be great to hear from other managed K8s providers if they are planning support

**Comment by bowei** [comment]
Check out the implementations page (https://gateway-api.sigs.k8s.io/implementations/) and see if any meet your needs.
